docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -121,6 +132,55 @@ async function renderApiDocumentation() {
 	console.groupEnd();
 }
 
+/**
+ * Processes documents and generates data required for the nav bar.
+ * @param {Array<Object>} documents - List of documents with apiItem.
+ * @param {ApiItem | undefined} documents.apiItem - The API item that the document is created from. Some documents may not have an apiItem.
+ */
+function buildNavBar(documents) {
```

Comment:
Nit: all of the other file-writing behavior in this build is async. I would recommend making this function async and using the async variants of writeFile, etc.